app/views/search/_granule_list.html.erb

```diff
@@ -1,76 +1,68 @@
       <section id="granule-list" class="master-overlay-no-min granule-list" data-bind="visible: ui.collectionsList.focused()" tabIndex="2" data-master-overlay-title="Granules">
         <nav class="master-overlay-nav">
           <a class="master-overlay-back master-overlay-hide-secondary" href="#" data-bind="click: function() {ui.projectList.hideFilters(); ui.collectionsList.unfocusCollection(); }">
-            <i class="fa fa-arrow-circle-o-left"></i> Back to Collections
+            <i class="fa fa-chevron-circle-left"></i> Back to Collections
           </a>
         </nav>
         <!-- ko if: ui.collectionsList.focused() && ui.collectionsList.focused().granules()  -->
         <!-- ko with: ui.collectionsList.focused() -->
         <header class="with-options">
-          <h2 data-bind="text: collection.title"></h2>
-          <!-- ko with: collection -->
+          <div class="collection-title-filters">
+            <h2><span data-bind="text: collection.title"></span> <!-- ko with: collection --><a href="#" class="master-overlay-forward collection-title-link" data-master-overlay-forward="Granules" data-bind="click: $root.ui.collectionsList.showCollectionDetails" title="View collection details">View details</a><!-- /ko --></h2>
```

Comment:
Any thoughts on including `fa-info-circle` with the "View details" link? That may help bridge the gap between old and new. I could see some people not catching the new way. Up to you, but worth asking.